lms/static/lms/js/require-config.js

```diff
@@ -75,7 +75,8 @@
             "catch": "js/vendor/ova/catch/js/catch",
             "handlebars": "js/vendor/ova/catch/js/handlebars-1.1.2",
             "tinymce": "js/vendor/tinymce/js/tinymce/tinymce.full.min",
-            "jquery.tinymce": "js/vendor/tinymce/js/tinymce/jquery.tinymce.min"
+            "jquery.tinymce": "js/vendor/tinymce/js/tinymce/jquery.tinymce.min",
+            "ccx_display_name": "js/ccx/ccx_display_name"
             // end of files needed by OVA
```

Comment:
We don't want our javascript to be associated with OVA. It's a dependency for an xmodule that's very likely to be deprecated soon.